gravitee-apim-console-webui/src/management/settings/groups/group/delete-member-dialog/delete-member-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title>Remove Member</h2>
+<mat-dialog-content>
+  @if (member.roles['API'] === 'PRIMARY_OWNER') {
+    <form [formGroup]="deleteMemberForm" class="delete-member__form">
+      <mat-form-field appearance="outline">
+        <mat-label>Search Members</mat-label>
+        <input aria-label="Search members" matInput formControlName="searchTerm" [matAutocomplete]="auto" />
+        <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selectPrimaryOwner($event)">
+          @for (member of filteredMembers$ | async; track member.id) {
+            <mat-option [value]="member">
+              {{ member.displayName }}
+            </mat-option>
+          }
+        </mat-autocomplete>
+        <mat-hint>Select a member to transfer primary ownership</mat-hint>
+      </mat-form-field>
+
+      @if (newPrimaryOwner) {
+        <mat-chip-set>
+          <mat-chip [removable]="true" (removed)="deselectPrimaryOwner()">
+            {{ newPrimaryOwner.displayName }}
+            <mat-icon matChipRemove>cancel</mat-icon>
+          </mat-chip>
+        </mat-chip-set>
+      }
+
+      @if (this.ownershipTransferMessage) {
+        <mat-hint>{{ this.ownershipTransferMessage }}</mat-hint>
```

Comment:
```suggestion
        <mat-hint>{{ ownershipTransferMessage }}</mat-hint>
```
It is not necessary to use `this` in the HTML template 👍